openghg/standardise/oned_timeseries/_crf.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+from pathlib import Path
+from typing import Dict, Optional
+
+
+def parse_crf(
+    filepath: Path,
+    species: str,
+    source: str = "crf",
+    domain: str = "europe",
+    data_type: str = "flux",
+    database: Optional[str] = None,
+    database_version: Optional[str] = None,
+    model: Optional[str] = None,
+    chunks: Optional[Dict] = None,
+) -> Dict:
+    """
+    Parse INTEM emissions data from the specified file.
```

Comment:
Should this say parse "CRF" emissions data? (Similarly in description of `filepath`)